stdlib/Dates/src/conversions.jl

```diff
@@ -38,26 +38,33 @@ Base.convert(::Type{Day},dt::Date) = Day(value(dt))            # Converts Date t
 
 ### External Conversions
 const UNIXEPOCH = value(DateTime(1970)) #Rata Die milliseconds for 1970-01-01T00:00:00
+const LOCALEPOCH = value(DateTime(Libc.TmStruct(0))) #Rata Die for local time at UTC 1970-01-01T00:00:00
```

Comment:
Making this a constant doesn't seem right — what if the local time changes?

Reply:
I would just make it a function
```jl
localepoch() = value(DateTime(Libc.TmStruct(0))) #Rata Die for local time at UTC 1970-01-01T00:00:00
```

Reply:
this makes sense, I did not think of this. Will make the changes accordingly.

Reply:
> Moreover, won't precompilation then set LOCALEPOCH to wherever Julia was built?

I thought it would be recalculated every time a function utilising `LOCALEPOCH` was called.